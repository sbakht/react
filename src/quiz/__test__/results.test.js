import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Results from '../results';
import toJSON from 'enzyme-to-json';

test('render default', () => {
  var questions = Array(4);
  var answers = [{isCorrect : true}, {isCorrect : true}] 
  var component = shallow(<Results questions={questions} answers={answers} />);
  expect(toJSON(component)).toMatchSnapshot();
});

test('has proptypes', () => {
  expect(Results.propTypes).toBeDefined();
  
  Object.keys(Results.propTypes).map((key) => {
    expect(Results.propTypes[key]).toBeDefined();
  });
});