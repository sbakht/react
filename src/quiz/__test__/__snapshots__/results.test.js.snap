// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render default 1`] = `
<div>
  Score: 
  2
  /
  4
</div>
`;
